pkg/util/kubernetes.go

```diff
@@ -161,6 +194,16 @@ func GetString(m map[string]string, key string) string {
 	return m[key]
 }
 
+func CreateInitContainer(r *api.Restic, tag string, workload api.LocalTypedReference, rbac bool) core.Container {
+	container := CreateSidecarContainer(r, tag, workload)
+	container.Args = append(container.Args, "--offline=true")
+	container.Args = append(container.Args, "--tag="+tag)
+	if rbac {
+		container.Args = append(container.Args, "--rbac=true")
+	}
+	return container
+}
+
 func CreateSidecarContainer(r *api.Restic, tag string, workload api.LocalTypedReference) core.Container {
```

Comment:
Where is serviceAccountName created

Reply:
InitContainer and Sidecar use the same service account as the workload?